src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+public class Event extends Task{
+    protected String start;
+    protected String end;
+
+    public Event(String description, String start, String end){
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
```

Comment:
Perhaps standardising the space between ) and { on line 5 would make things look neater. I noticed this across multiple files as well.
```suggestion
    public Event(String description, String start, String end) {
        super(description);
        this.start = start;
        this.end = end;
    }
```